logic/common/log_utils.py

```diff
@@ -0,0 +1,112 @@
+''' Utility functions for printing and logging
+
+USAGE EXAMPLE:
+
+import logic.common.utils_log as log
+    log.SetVerbosityLevel(1)
+    log.Info("Hello World!")
+
+# Verbosity level is usually first set in CLI file once during initialisation
+# Afterwards, you may use log.Info(s), log.Must(s), log.Extra(s) anywhere you need
+
+'''
+
+import os
+import logging
+
+#--------------------------------------------------#
+'''Logger Settings'''
+
+DEFAULT_FILENAME = 'logfile.log'
+DEFAULT_FORMAT = '%(message)s'   # Without extra indentation
+DEFAULT_FORMAT = '\t%(message)s' # With extra indentation
```

Comment:
I sometimes place the definitions this way when there are multiple "options". For example, when we have only a single line like `x = 101` and want to change the value, we would need to select the number, delete, then re-enter a new value, e.g. `x = 256`. But if we put things into multiple lines like this:

```
x = 101
x = 256
```

I can just swap the two lines by doing [Shift]+[UpArrow] -> [Cut] -> [UpArrow] -> [Paste], regardless what the format or value the statements have. This is much quicker for me through muscle memory (than having to think what the new value should be), since it's the same keypresses regardless.

As for the example of changing `DEFAULT_FORMAT` here, we can easily swap between 2 format based on user's preference. For instance, I would keep the one with `\t` at bottom since I prefer the extra indentation, but you can swap the order if you prefer not having said indentation. The added benefit is that it also retains the comment stating what the current option does, without removing other possible options (& comment) in case we want to change it to something else later.

Anyway, I'll be removing this if you dislike it. Just want to explain why these seemingly pointless lines exist.

--------------------------------------------------

Just some food for thoughts, but there are other potentially useful options too.

```
DEFAULT_FORMAT = '    %(message)s'         # w/ 4-space indentation
DEFAULT_FORMAT = '(datetime)\t%(message)s' # Included logging time
```